src/app/donation-complete/donation-complete.component.ts

```diff
@@ -219,8 +219,8 @@ export class DonationCompleteComponent implements OnInit {
     this.campaignService.getOneById(donation.projectId).subscribe(campaign => {
       this.campaign = campaign;
       this.pageMeta.setCommon(
-        `${campaign.charity.name}`,
-        `Thanks for donating to the "${campaign.title}" campaign`,
+        `Thank you for donating to the "${campaign.title}" campaign`,
```

Comment:
I was hoping this might be auto read by the screen reader. It doesn't seem to be me for me. Still we might as well have "Thank you" in the title.